aspnetcore/security/docker-compose-https.md

```diff
@@ -99,7 +99,7 @@ dotnet dev-certs https -ep ${HOME}/.aspnet/https/aspnetapp.pfx -p $CREDENTIAL_PL
 dotnet dev-certs https --trust
 ```
 
-`dotnet dev-certs https --trust` is only supported on macOS and Windows. You need to trust certificates on Linux in the way that is supported by your distribution. It is likely that you need to trust the certificate in your browser.
+Prior to 9.0, `dotnet dev-certs https --trust` is only supported on macOS and Windows. You need to trust certificates on Linux in the way that is supported by your distribution. It is likely that you need to trust the certificate in your browser.
```

Comment:
```suggestion
On Linux, `dotnet dev-certs https --trust` requires .NET 9 and later. For Linux on .NET 8 and earlier, see your Linux distribution's documentation for trusting a certificate.
```

Reply:
> It is likely that you need to trust the certificate in your browser.

@amcasey  I can add that back but it seems obvious.

Reply:
Personally, I was surprised to learn that browsers don't respect system trust on Linux and require separate configuration.  However, it's possible that regular Linux users expect that.